layouts/partials/head.html

```diff
@@ -27,4 +27,7 @@
 
   <!-- RSS -->
   <link href="{{ .RSSLink }}" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
+
+  <!-- Hook for custom head matter -->
```

Comment:
I don't think people would want this comment in their HTML header, as it will only make sense if the partal below actual exists.

If you want an "invisible" comment, use:

```bash
{{/* some comment */}}
``

Reply:
Thank you for your patience with me. I'll just take the comment out.